kroxylicious-operator/pom.xml

```diff
@@ -296,103 +280,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
Is there a reason why the dependency analysis shouldn't be done in all the modules?

Reply:
It is by default IIRC, I don't think we need to customise it in the operator any longer.

I'll double check that is true 😁

Reply:
Confirmed, dependency analysis is run by default.